51.md

```diff
@@ -6,148 +6,128 @@ Lists
+    ["p", "a55c15f5e41d5aebd236eca5e0142789c5385703f1a7485aa4b38d94fd18dcc4"]
   ],
-  "content": "VezuSvWak++ASjFMRqBPWS3mK5pZ0vRLL325iuIL4S+r8n9z+DuMau5vMElz1tGC/UqCDmbzE2kwplafaFo/FnIZMdEj4pdxgptyBV1ifZpH3TEF6OMjEtqbYRRqnxgIXsuOSXaerWgpi0pm+raHQPseoELQI/SZ1cvtFqEUCXdXpa5AYaSd+quEuthAEw7V1jP+5TDRCEC8jiLosBVhCtaPpLcrm8HydMYJ2XB6Ixs=?iv=/rtV49RFm0XyFEwG62Eo9A==",
-  ...other fields
+  "content": "TJob1dQrf2ndsmdbeGU+05HT5GMnBSx3fx8QdDY/g3NvCa7klfzgaQCmRZuo1d3WQjHDOjzSY1+MgTK5WjewFFumCcOZniWtOMSga9tJk1ky00tLoUUzyLnb1v9x95h/iT/KpkICJyAwUZ+LoJBUzLrK52wNTMt8M5jSLvCkRx8C0BmEwA/00pjOp4eRndy19H4WUUehhjfV2/VV/k4hMAjJ7Bb5Hp9xdmzmCLX9+64+MyeIQQjQAHPj8dkSsRahP7KS3MgMpjaF8nL48Bg5suZMxJayXGVp3BLtgRZx5z5nOk9xyrYk+71e2tnP9IDvSMkiSe76BcMct+m7kGVrRcavDI4n62goNNh25IpghT+a1OjjkpXt9me5wmaL7fxffV1pchdm+A7KJKIUU3kLC7QbUifF22EucRA9xiEyxETusNludBXN24O3llTbOy4vYFsq35BeZl4v1Cse7n2htZicVkItMz3wjzj1q1I1VqbnorNXFgllkRZn4/YXfTG/RMnoK/bDogRapOV+XToZ+IvsN0BqwKSUDx+ydKpci6htDRF2WDRkU+VQMqwM0CoLzy2H6A2cqyMMMD9SLRRzBg==?iv=S3rFeFr1gsYqmQA7bNnNTQ==",
+  "sig": "1173822c53261f8cffe7efbf43ba4a97a9198b3e402c2a1df130f42a8985a2d0d3430f4de350db184141e45ca844ab4e5364ea80f11d720e36357e1853dba6ca"
 }
 ```
 
-Lets say a user wants to create a 'Categorized Bookmarks' list of `bookmarks` and has keys:
-```
-priv: fb505c65d4df950f5d28c9e4d285ee12ffaf315deef1fc24e3c7cd1e7e35f2b1
-pub: b1a5c93edcc8d586566fde53a20bdb50049a97b15483cb763854e57016e0fa3d
-```
-The user wants to publicly include these bookmarks:
+### A _curation set_ of articles and notes about yaks
 
-```json
-["e", "5c83da77af1dec6d7289834998ad7aafbd9e2191396d75ec3cc27f5a77226f36", "wss://nostr.example.com"],
-["a", "30023:f7234bd4c1394dda46d09f35bd384dd30cc552ad5541990f98844fb06676e9ca:abcd", "wss://nostr.example.com"],
-["r", "https://github.com/nostr-protocol/nostr", "Nostr repository"],
 ```
-and privately include these bookmarks (below is the JSON that would be encrypted and placed in the event content):
-
-```json
-[
-    ["r", "https://my-private.bookmark", "My private bookmark"],
-    ["a", "30001:f7234bd4c1394dda46d09f35bd384dd30cc552ad5541990f98844fb06676e9ca:abcd", "wss://nostr.example.com"],
-]
-```
-
-Then the user would create a 'Categorized Bookmarks' list event like below:
-
-```json
 {
-  "kind": 30001,
+  "id": "567b41fc9060c758c4216fe5f8d3df7c57daad7ae757fa4606f0c39d4dd220ef",
+  "pubkey": "d6dc95542e18b8b7aec2f14610f55c335abebec76f3db9e58c254661d0593a0c",
+  "created_at": 1695327657,
+  "kind": 30004,
   "tags": [
-    ["d", "bookmarks"],
-    ["e", "5c83da77af1dec6d7289834998ad7aafbd9e2191396d75ec3cc27f5a77226f36", "wss://nostr.example.com"],
-    ["a", "30023:f7234bd4c1394dda46d09f35bd384dd30cc552ad5541990f98844fb06676e9ca:abcd", "wss://nostr.example.com"],
-    ["r", "https://github.com/nostr-protocol/nostr", "Nostr repository"],
+    [
+      "d",
+      "jvdy9i4"
+    ],
+    [
+      "title",
+      "Yaks"
+    ],
+    [
+      "summary",
```

Comment:
is it supposed to be a "summary" or "description" tag? because earlier on line 50 it says "title", "image", and "description" tags

Reply:
Good question. What should it be?

Reply:
I think it's better to go with `name`, `picture` and `about`, following kind 0 nomenclature.

Reply:
I believe listr.lol is already using "description" but I'm fine with any of them as long as there is a set standard
"about" or "summary" might be slightly better working since they imply a single line of text and not a full paragraph

Reply:
I'm using `title` and `description` tags on lists in Listr. I would love to leave those be, as they seem more applicable than `name`, `about`. I would also be in favor of using `image` instead of `picture`, keeping more in line with NIP-23 than kind 0. 

FWIW, the reason for the `title` tag is that in order to give users the ability to edit the name of a list without creating a brand new list, you have to have to keep the `d` tag constant, so I use a uuid value for the `d` tag in Listr and then show the `title` tag in the UI instead of the `d` tag. This pattern works really well for user defined lists and means that other clients can also allow users to rename their lists, as long as they respect and maintain the `d` tag.

Reply:
> I'm using `title` and `description` tags on lists in Listr. I would love to leave those be, as they seem more applicable than `name`, `about`. I would also be in favor of using `image` instead of `picture`, keeping more in line with NIP-23 than kind 0.
> 
> FWIW, the reason for the `title` tag is that in order to give users the ability to edit the name of a list without creating a brand new list, you have to have to keep the `d` tag constant, so I use a uuid value for the `d` tag in Listr and then show the `title` tag in the UI instead of the `d` tag. This pattern works really well for user defined lists and means that other clients can also allow users to rename their lists, as long as they respect and maintain the `d` tag.

We're using the `d` tag for the same exact reason allowing editing while maintaining the same list.

Reply:
> I'm using `title` and `description` tags on lists in Listr. I would love to leave those be, as they seem more applicable than `name`, `about`. I would also be in favor of using `image` instead of `picture`, keeping more in line with NIP-23 than kind 0.

I think it is better to keep the generic tags for `kind-30004` to `title`, `description`, and `image` or `thumbnail` since it is widely used same as @erskingardner suggested.

Reply:
changed to `title`, `image` and `description`.